### DISCUSSION TITLE
Filament Table Filter: How to remove "Filters" text?

### DISCUSSION THREAD

**Discussion by devhoussam1998** [filamentphp/filament#12112] [Help] [answered]
### Package

Table builder

### Package Version

v3.2.61

### How can we help you?

I want to remove **Filters** text.

<img width="893" alt="Screenshot 2024-03-30 at 17 41 07" src="https://github.com/filamentphp/filament/assets/121255432/95477da9-e07a-4e30-b053-1d3bc81665be">

**Comment by leandrocfe** [comment, marked as answer]
Publish the table translation: `php artisan vendor:publish --tag=filament-tables-translations`
`lang/vendor/filament-tables/en/table.php`
```php
'heading' => 'Filters' //probably
```

**Reply by mikailhamzat** [reply]
This probably works but is there a way to edit per resource?
